scss/components/_search-bar.scss

```diff
@@ -76,7 +78,7 @@
     }
 
     .combo__button {
-      width: u(5.6rem);
+      width: #{$search-button-width};
```

Comment:
Shouldn't need the string interpolation here, you can just use the variable directly:
```
width: $search-button-width;
```